internals/demo.sh

```diff
@@ -19,15 +32,21 @@ DEMO_PROMPT="$BLUE~/demo-repository$COLOR_RESET$ "
 ### initial setup ###
 
 p "git init"
-git init > /dev/null
+git init &> /dev/null
```

Comment:
To be honest, I would rather love to have errors be displayed in the demo.
If something fails here a user would have no indication that it went wrong.

WDYT?

Reply:
Yes I was thinking about that as well. I guess it's better to leave the error output on.
I only saw the very wordy default error about the branch name. But it's better to fix the root-cause (the branch name) than to hide potential other errors.